package-lock.json

```diff
@@ -55113,7 +55113,7 @@
         "@babel/core": "^7.14.8",
         "@babel/eslint-parser": "^7.13.14",
         "@babel/preset-env": "^7.14.8",
-        "@kiva/kv-tokens": "^2.12.0",
+        "@kiva/kv-tokens": "*",
```

Comment:
What caused this change? Should it be reverted?

Reply:
Oh, I didn't see that. Just updated the package as usual.
I guess should be reverted...